### PR TITLE
add config `shutdown.wait` and update version to 0.7.0-SNAPSHOT

### LINKED ISSUE #1212: when destroy server, lack config `transport.shutdown.wait`
- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description

when destroy server, lack config `transport.shutdown.wait`

### Ⅱ. Describe what happened

  
image

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. xxx
2. xxx
3. xxx

### Ⅵ. Environment:

- JDK version :
- OS :
- Others:

### PR DESCRIPTION
**Pull request by slievrly**
Signed-off-by: slievrly <slievrly@163.com>

### Ⅰ. Describe what this PR did

add config `shutdown.wait` and update version to 0.7.1-SNAPSHOT

### Ⅱ. Does this pull request fix one issue?
fix #1212

### CODE REVIEW

**Comment by xingfudeshi** [comment]
In the tiltle,the version may be 0.7.0-SNAPSHOT?

File: core/src/main/java/io/seata/core/protocol/Version.java [hunk 1]
```diff
@@ -31,7 +31,7 @@
     /**
      * The constant CURRENT.
      */
-    public static final String CURRENT = "0.6.1";
+    public static final String CURRENT = "0.7.0";
```
Comment (lovepoem) [inline core/src/main/java/io/seata/core/protocol/Version.java]: How about use the version configured in pom.xml  dynamicly?

Comment (lovepoem) [reply core/src/main/java/io/seata/core/protocol/Version.java]: or the version in meta-info in the jar ?

Comment (slievrly) [reply core/src/main/java/io/seata/core/protocol/Version.java]: But when we run with IDE, there is no meta-info here.

Comment (lovepoem) [reply core/src/main/java/io/seata/core/protocol/Version.java]: > But when we run with IDE, there is no meta-info here.

I think it is a question , we can retain this issue temporarily and merge this pr

**Pull request merged**